smart_card_connector_app/src/application_unittest.cc

```diff
@@ -2466,6 +2466,37 @@ TEST_F(SmartCardConnectorApplicationSingleClientTest,
   EXPECT_EQ(new_active_protocol, active_protocol);
 }
 
+// `SCardReconnect()` call from JS succeeds when using the same parameters as
+// the previous `SCardConnect()` call.
+TEST_F(SmartCardConnectorApplicationConnectedReaderTest, SCardReconnectReset) {
+  // Reconnect using the same sharing and protocol as the `SCardConnect()` call
+  // in the test's setup.
+  DWORD new_active_protocol = 0;
+  LONG return_code = SimulateReconnectCallFromJsClient(
+      kFakeHandlerId, scard_handle(), SCARD_SHARE_SHARED,
+      /*preferred_protocols=*/SCARD_PROTOCOL_T1, SCARD_RESET_CARD,
+      new_active_protocol);
+
+  EXPECT_EQ(return_code, SCARD_S_SUCCESS);
+  EXPECT_EQ(new_active_protocol, static_cast<DWORD>(SCARD_PROTOCOL_T1));
+}
+
+// `SCardReconnect()` call from JS succeeds when using the same parameters as
```

Comment:
I think the two comments of the tests should not be identical, but call out the difference in the tests.

Reply:
done, thanks for catching this copy-paste error